src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball {
+    private Color color = ColorSupplier.getRandomColor();
+    private int number = new Random().nextInt(100);
```

Comment:
what if we want to create a ball with some specific color and number?
(it would be better to have a constructor for this fields)